sql/item_weapon.sql

```diff
@@ -1116,54 +1113,54 @@ INSERT INTO `item_weapon` VALUES (17468,'raise_rod',11,0,0,0,0,3,1,288,10,0);
-INSERT INTO `item_weapon` VALUES (17487,'corrosive_claws',1,0,0,0,0,4,1,297,13,0);
-INSERT INTO `item_weapon` VALUES (17488,'cermet_claws_+1',1,0,0,0,0,4,1,282,15,0);
-INSERT INTO `item_weapon` VALUES (17489,'gold_patas_+1',1,0,0,0,0,4,1,346,22,0);
-INSERT INTO `item_weapon` VALUES (17490,'feral_fangs',1,0,0,0,0,4,1,336,22,0);
-INSERT INTO `item_weapon` VALUES (17491,'spartan_cesti',1,0,0,0,0,4,1,353,23,0);
-INSERT INTO `item_weapon` VALUES (17492,'shivas_claws',1,0,0,0,0,4,1,300,14,0);
-INSERT INTO `item_weapon` VALUES (17493,'tropical_punches',1,0,0,0,0,4,1,240,4,0);
-INSERT INTO `item_weapon` VALUES (17494,'trp._punches_+1',1,0,0,0,0,4,1,240,5,0);
-INSERT INTO `item_weapon` VALUES (17495,'san_dorian_cesti',1,0,0,0,0,4,1,278,6,0);
-INSERT INTO `item_weapon` VALUES (17496,'kingdom_cesti',1,0,0,0,0,4,1,273,7,0);
-INSERT INTO `item_weapon` VALUES (17497,'win._baghnakhs',1,0,0,0,0,4,1,297,7,0);
-INSERT INTO `item_weapon` VALUES (17498,'fed._baghnakhs',1,0,0,0,0,4,1,288,8,0);
-INSERT INTO `item_weapon` VALUES (17499,'bas._knuckles',1,0,0,0,0,4,1,326,8,0);
-INSERT INTO `item_weapon` VALUES (17500,'republic_knuckles',1,0,0,0,0,4,1,316,9,0);
-INSERT INTO `item_weapon` VALUES (17501,'t.m._hooks_+1',1,0,0,0,0,4,1,321,18,0);
-INSERT INTO `item_weapon` VALUES (17502,'t.m._hooks_+2',1,0,0,0,0,4,1,311,19,0);
-INSERT INTO `item_weapon` VALUES (17503,'exocets',1,0,0,0,0,4,1,291,13,0);
-INSERT INTO `item_weapon` VALUES (17504,'pagures',1,0,0,0,0,4,1,346,17,0);
-INSERT INTO `item_weapon` VALUES (17505,'narasimhas_cesti',1,0,0,0,0,4,1,240,9,0);
-INSERT INTO `item_weapon` VALUES (17506,'vishnus_cesti',1,0,0,0,0,4,1,240,10,0);
-INSERT INTO `item_weapon` VALUES (17507,'knuckles_of_trials',1,0,0,0,0,4,1,336,19,300);
-INSERT INTO `item_weapon` VALUES (17508,'mst.cst.baghnakhs',1,0,0,0,0,4,1,306,18,0);
-INSERT INTO `item_weapon` VALUES (17509,'destroyers',1,0,0,0,0,4,1,288,8,500);
-INSERT INTO `item_weapon` VALUES (17510,'vampiric_claws',1,0,0,0,0,2,1,300,11,0);
-INSERT INTO `item_weapon` VALUES (17511,'indra_katars',1,0,0,0,0,4,1,324,22,0);
-INSERT INTO `item_weapon` VALUES (17512,'hydro_baghnakhs',1,0,0,0,0,4,1,300,7,0);
-INSERT INTO `item_weapon` VALUES (17513,'hyd.baghnakhs_+1',1,0,0,0,0,4,1,295,8,0);
-INSERT INTO `item_weapon` VALUES (17514,'hydro_claws',1,0,0,0,0,4,1,306,10,0);
-INSERT INTO `item_weapon` VALUES (17515,'hydro_claws_+1',1,0,0,0,0,4,1,301,11,0);
-INSERT INTO `item_weapon` VALUES (17516,'hydro_patas',1,0,0,0,0,4,1,336,19,0);
-INSERT INTO `item_weapon` VALUES (17517,'hydro_patas_+1',1,0,0,0,0,4,1,326,20,0);
-INSERT INTO `item_weapon` VALUES (17518,'manoples_+1',1,0,0,0,0,4,1,346,24,0);
-INSERT INTO `item_weapon` VALUES (17519,'platoon_cesti',1,0,0,0,0,4,1,288,7,0);
+INSERT INTO `item_weapon` VALUES (17472,'cross-counters',1,0,0,0,0,4,1,576,22,0);
+INSERT INTO `item_weapon` VALUES (17473,'torama_cesti',1,0,0,0,0,4,1,520,9,0);
+INSERT INTO `item_weapon` VALUES (17474,'grapnel',0,0,0,0,0,0,1,0,3,0);
```

Comment:
No big deal and yes this matches short name from item basic

On Saturday, May 14, 2016, TeoTwawki <notifications@github.com> wrote:

> In sql/item_weapon.sql
> <https://github.com/DarkstarProject/darkstar/pull/3170#discussion_r63280145>
> :
>
> > -INSERT INTO `item_weapon` VALUES (17507,'knuckles_of_trials',1,0,0,0,0,4,1,336,19,300);
> > -INSERT INTO `item_weapon` VALUES (17508,'mst.cst.baghnakhs',1,0,0,0,0,4,1,306,18,0);
> > -INSERT INTO `item_weapon` VALUES (17509,'destroyers',1,0,0,0,0,4,1,288,8,500);
> > -INSERT INTO `item_weapon` VALUES (17510,'vampiric_claws',1,0,0,0,0,2,1,300,11,0);
> > -INSERT INTO `item_weapon` VALUES (17511,'indra_katars',1,0,0,0,0,4,1,324,22,0);
> > -INSERT INTO `item_weapon` VALUES (17512,'hydro_baghnakhs',1,0,0,0,0,4,1,300,7,0);
> > -INSERT INTO `item_weapon` VALUES (17513,'hyd.baghnakhs_+1',1,0,0,0,0,4,1,295,8,0);
> > -INSERT INTO `item_weapon` VALUES (17514,'hydro_claws',1,0,0,0,0,4,1,306,10,0);
> > -INSERT INTO `item_weapon` VALUES (17515,'hydro_claws_+1',1,0,0,0,0,4,1,301,11,0);
> > -INSERT INTO `item_weapon` VALUES (17516,'hydro_patas',1,0,0,0,0,4,1,336,19,0);
> > -INSERT INTO `item_weapon` VALUES (17517,'hydro_patas_+1',1,0,0,0,0,4,1,326,20,0);
> > -INSERT INTO `item_weapon` VALUES (17518,'manoples_+1',1,0,0,0,0,4,1,346,24,0);
> > -INSERT INTO `item_weapon` VALUES (17519,'platoon_cesti',1,0,0,0,0,4,1,288,7,0);
> > +INSERT INTO `item_weapon` VALUES (17472,'cross-counters',1,0,0,0,0,4,1,576,22,0);
> > +INSERT INTO `item_weapon` VALUES (17473,'torama_cesti',1,0,0,0,0,4,1,520,9,0);
> > +INSERT INTO `item_weapon` VALUES (17474,'grapnel',0,0,0,0,0,0,1,0,3,0);
>
> Pardon, only half awake. These are matching the short-name in item basic
> yeah? They are fine and I don't think we even use this name column in
> item_weapon (not sure). The long name in item_basic is the one that
> matters, some folk were annoyed at colon's in there the other day so I
> remembered it. I'll stfu now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/DarkstarProject/darkstar/pull/3170/files/757697918a9cb38768e92bbd2be1d399d2269f31#r63280145>
>